data/abilities/basic_attack.json

```diff
@@ -1,5 +1,7 @@
 {
     "name": "Attack",
+    "description": "Simple basic attack available to all golems",
```

Comment:
Not all golems actually have access to this. The current job determines which ability gets used as a "basic attack."

Reply:
Oh shoot, you're right!